spec/requests/health_care_applications_request_spec.rb

```diff
@@ -476,9 +480,34 @@ def self.expect_async_submit
 
         it 'does not error on vaMedicalFacility validation' do
           subject
+
+          expect(JSON.parse(response.body)['errors']).to be_blank
           expect(JSON.parse(response.body)['data']['attributes']).to eq(body)
         end
       end
+
+      context 'with hca_use_facilities_API disabled' do
```

Comment:
Woohoo. Now it fails for me when I comment it out because of this test. Nice work!

Reply:
TDD     Constructive Sabotage
        🤝 
"break it til it's fixed!"